src/nemo-location-entry.c

```diff
@@ -330,7 +330,9 @@ nemo_location_entry_activate (GtkEntry *entry)
 	if (entry_text != NULL && *entry_text != '\0') {
 		uri_scheme = g_uri_parse_scheme (entry_text);
 
```

Comment:
I think we should trim leading and trailing space before we actually handle it using something like `g_strstrip()` - then I think we can just let it be processed like before.